### ISSUE TITLE
autoprefixer not working, although it's required in webpack.common

### ISSUE THREAD

**Issue by ayazhussein** [angular/angular-cli#3123]
autoprefixer is required in webpack common and it's not working, even if I specify a browserlist.

**Comment by Meligy** [comment]
This is a duplicate of #1512, which was fixed with PR #2852, which is not released yet.

Should be fixed in the upcoming release.

**Comment by filipesilva** [comment]
Should be present in the latest release.

**Comment by nnennajohn** [comment]
HI filipesilva, can you please add to the documentation on how to use it? Thanks.

**Comment by marcalj** [comment]
Yes please, add documentation, I couldn't find how I can configure options :/
An example of custom config would be enough. Thanks!

**Comment by btroncone** [comment]
Any documentation or examples on this? Thanks in advance!

**Comment by btroncone** [comment]
@marcalj It looks like this is included by default, no config required.

**Comment by marcalj** [comment]
@btroncone But, if I want to define a config for custom IE version, how I have to do it?
I've created a ```browserslist``` file in ng-cli's project root, with:

```
Last 2 versions
IE 11
```

It's ok? I've just guess it through stack overflow and issue tickets, not from official wiki or README. Thanks.
